docs/ops/normalization/NormalizeL2_1.md

```diff
@@ -4,7 +4,22 @@
 
 **Category**: *Normalization*
 
-**Short description**: *NormalizeL2* operation performs L2 normalization of the 1st input tensor in slices specified by the 2nd input.
+**Short description**: *NormalizeL2* operation performs L2 normalization of the first input tensor in slices specified by the second input.
```

Comment:
Personally I like to indicate names of inputs if there is more than 1. In addition, I think we can omit tensor here as in inputs/outputs sections specify that inputs and outputs are tensors. Similarly as in [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/reduction/ReduceL2_4.md).

Something like:
performs L2 normalization on a given input \`data\` along dimensions specified by \`axes\` input

Reply:
Done